scripts/station_csv_to_json.py

```diff
@@ -14,8 +15,15 @@
     # Keep track of station count for debug purposes.
     station_count = 0
     for row in reader:
-        # We're only interested in active weather stations.
+        # We're only interested in permanent, active weather stations
+        # Active stations are either marked as 'active' in the station_category row,
+        # or start with ZZ.
```

Comment:
other way around - station names that start with ZZ are archived. Your code is removing the ones that start with ZZ, but I recommend correcting this comment to avoid confusion later on